fastlane/Fastfile

```diff
@@ -34,7 +34,7 @@ platform :ios do
   desc "Deploy to MS App-Center"
   lane :appcenter do
     match(
-      app_identifier: ['com.ecosia.ecosiaapp.firefox', 'com.ecosia.ecosiaapp.firefox.searchWidget', 'com.ecosia.ecosiaapp.firefox.WidgetKit', 'com.ecosia.ecosiaapp.firefox.ShareTo'], 
+      app_identifier: ['com.ecosia.ecosiaapp.firefox', 'com.ecosia.ecosiaapp.firefox.searchWidget', 'com.ecosia.ecosiaapp.firefox.WidgetKit', 'com.ecosia.ecosiaapp.firefox.ShareTo', 'com.ecosia.ecosiaapp.firefox.CredentialProvider'], 
```

Comment:
Interesting this new CredentialProvider 👍

Reply:
@vauxhall Yes indeed. I am not 100% sure what this extension is actually doing. It suspect it provides access to our stored credenitals for other apps. E.g. when in Safari and a password needs to be filled, there is some UI which will give you access to the FF credentials. But I haven't found a way to trigger it. Happy to gain any insights if you have any :)

Reply:
Or it provides access to iOS' stored credentials to be used inside the app 🤔 as said needs some investigation

Reply:
FYI: that's the related PR: https://github.com/mozilla-mobile/firefox-ios/pull/8683